### PR TITLE
Fixes #4512 - Add line and link to about_PSReadline

### LINKED ISSUE #4512: Suggest adding "Addline" and its reference to this article(about_Line_Editing)
The [addline](https://docs.microsoft.com/en-us/powershell/module/psreadline/about/about_psreadline?view=powershell-5.1#addline) is needed to be listed for editing in the PowerShell command prompt. Suggest adding [about_psreadline](https://docs.microsoft.com/en-us/powershell/module/psreadline/about/about_psreadline?view=powershell-5.1#addline) as a reference to "SEE ALSO" list.

this is also mentioned in [ISE # Command Pane](https://docs.microsoft.com/en-us/powershell/scripting/components/ise/exploring-the-windows-powershell-ise?view=powershell-5.1#command-pane) what is alomost the same as PowerShell command prompt.

> Press SHIFT+ENTER to enter each line of a multiline command, and press ENTER after the last line to execute the multiline command 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 8a747f17-9e1d-0993-44eb-ffc906600d28
* Version Independent ID: 7d359679-1451-7db3-6367-6103c046b3a1
* Content: [about_Line_Editing](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_line_editing?view=powershell-5.1)
* Content Source: [reference/5.1/Microsoft.PowerShell.Core/About/about_Line_Editing.md](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/5.1/Microsoft.PowerShell.Core/About/about_Line_Editing.md)
* Product: **powershell**
* Technology: **powershell-cmdlets**
* GitHub Login: @SteveL-MSFT
* Microsoft Alias: **slee**

### PR DESCRIPTION
**Pull request by davidsmatlak**
Version(s) of document impacted
------------------------------
- [x] Impacts 7 document
- [x] Impacts 6 document
- [x] Impacts 5.1 document
- [x] Impacts 5.0 document
- [x] Impacts 4.0 document
- [x] Impacts 3.0 document

about_Line_Editing Acrolinx score
Original version: 96
Updated version: 100

- Fixes [AB#1565857](https://dev.azure.com/mseng/677da0fb-b067-4f77-b89b-f32c12bb8617/_workitems/edit/1565857)
- Copyedits, style
- Added about_PSReadline link to v5.0 - 7

### CODE REVIEW

File: reference/3.0/Microsoft.PowerShell.Core/About/about_Line_Editing.md [hunk 1]
```diff
@@ -8,49 +8,74 @@ title:  about_Line_Editing
...
-## LONG DESCRIPTION
+## Long description
 
-The Windows PowerShell console has some useful features to help
-you to edit commands at the Windows PowerShell command prompt.
+The PowerShell console has some useful keyboard shortcuts to help you edit
+commands at the PowerShell command prompt.
 
-### Move Left and Right
+### Add a line
```
Comment (sdwheeler) [inline reference/3.0/Microsoft.PowerShell.Core/About/about_Line_Editing.md]: This does not apply to the version of PSReadLine that shipped in v3.

File: reference/4.0/Microsoft.PowerShell.Core/About/about_Line_Editing.md [hunk 2]
```diff
@@ -8,48 +8,73 @@ title:  about_Line_Editing
...
-## LONG DESCRIPTION
+## Long description
 
-The Windows PowerShell console has some useful features to help
-you to edit commands at the Windows PowerShell command prompt.
+The PowerShell console has some useful keyboard shortcuts to help you edit
+commands at the PowerShell command prompt.
 
-### Move Left and Right
+### Add a line
```
Comment (sdwheeler) [inline reference/4.0/Microsoft.PowerShell.Core/About/about_Line_Editing.md]: This does not apply to the version of PSReadLine that shipped in v4.

File: reference/3.0/Microsoft.PowerShell.Core/About/about_Line_Editing.md (hunk 1 above)
Comment (davidsmatlak) [reply reference/3.0/Microsoft.PowerShell.Core/About/about_Line_Editing.md]: Fixed.

File: reference/4.0/Microsoft.PowerShell.Core/About/about_Line_Editing.md (hunk 2 above)
Comment (davidsmatlak) [reply reference/4.0/Microsoft.PowerShell.Core/About/about_Line_Editing.md]: Fixed.

**Pull request merged**